server/app.ts

```diff
@@ -3,6 +3,7 @@ import { APIGatewayEvent, APIGatewayProxyCallback, Context } from 'aws-lambda';
 import { handler as getUser } from './lib/user/controller';
 import { handler as postUser } from './lib/user/post/controller';
 import { handler as createExpense } from './lib/expense/controller';
+import { getExpensesHandler } from "./lib/expense/controller";
```

Comment:
Import should look like this: import { handler as getExpensesHandler } from "./lib/expense/controller";